jaxrs-clients/src/main/java/com/palantir/remoting1/jaxrs/FeignJaxRsClientBuilder.java

```diff
@@ -163,7 +163,7 @@ private Decoder createDecoder(ObjectMapper objectMapper) {
                 .randomGenerator(new Random())
                 .state(new ThreadLocalServerClientAndLocalSpanState(
                         getIpAddress(), 0 /** Client TCP port. */, userAgent))
-                .spanCollector(new SlfLoggingSpanCollector("ClientTracer(" + userAgent + ")"))
+                .spanCollector(new SlfLoggingSpanCollector("tracing.client." + userAgent))
```

Comment:
I don't know. Either way, well need a more sanitized name.

On Tue, Aug 30, 2016, 12:00 David Schlosnagle <notifications@github.com>
wrote:

> In
> jaxrs-clients/src/main/java/com/palantir/remoting1/jaxrs/FeignJaxRsClientBuilder.java
> <https://github.com/palantir/http-remoting/pull/223#discussion_r76858288>:
>
> > @@ -163,7 +163,7 @@ private Decoder createDecoder(ObjectMapper objectMapper) {
> >                  .randomGenerator(new Random())
> >                  .state(new ThreadLocalServerClientAndLocalSpanState(
> >                          getIpAddress(), 0 /** Client TCP port. */, userAgent))
> > -                .spanCollector(new SlfLoggingSpanCollector("ClientTracer(" + userAgent + ")"))
> > +                .spanCollector(new SlfLoggingSpanCollector("tracing.client." + userAgent))
>
> How do you adjust the log config for the ClientTracer(com.palantir.product
> form? For tracing.client.com.palantir.product you can set it as tracing:
> INFO and send it to the desired appender.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/palantir/http-remoting/pull/223/files/38a16f3df23e07ede8a2f95cb309b0c5d045a0c3#r76858288>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGOdwTgzJJoIZGEiHjOJuLf3yk4bh00jks5qlH2tgaJpZM4JwrM1>
> .
>